### PR TITLE
docs(); modified (maple, alpha finance, cbridge and synapse)

### PR DESCRIPTION
**Pull request by ishraq8**
1. Error corrected in Maple finance for TVL calculation
2. Readme docs added for Alpha Finance, cbridge and synapse. 
3. Minor spelling and directory changes in these docs after feedback on how to correct. 

Let me know if there are any changes that need to be made.

### CODE REVIEW

**Comment by bye43** [comment]
@ishraq8 Docs look great! For the bridge docs, you may want to have @jaimehgb take a quick look since he is currently implementing Synapse.

**Review by bye43 — APPROVED** [review]

File: subgraphs/cbridge/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+cBridge introduces the best-in-class cross-chain token bridging experience with deep liquidity for users, highly efficient and easy-to-use liquidity management for both cBridge node operators and Liquidity Providers who do not want to operate cBridge nodes, and new exciting developer-oriented features such as general message bridging for cases like cross-chain DEX and NFTs. 
+
+## Usage Metrics
+
+`Active Users`, `Total Unique Users` & `Daily Transaction Count`
+
+Transactions of interest include:
+Cross-chain transfer of Tokens
+Cross-chain transfer of NFTs
```
Comment (jaimehgb) [inline subgraphs/cbridge/README.md:11]: The schema currently doesn’t support NFT transfers out of the box. But we can still count them as transactions / messages. And keep track of the users using that feature. Let’s merge this as it is, and if we discard this information I’ll amend it

File: subgraphs/synapse/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+Users of Synapse are those taking the following actions:
+
+Bridge Assets
+Swap Assets
+Liquidity Pools
+Deposit Assets
+Withdraw Assets
+Stake LP position
+
```
Comment (jaimehgb) [inline subgraphs/synapse/README.md:17]: Can also send messages through the bridge (currently only on certain chains)

https://github.com/synapsecns/synapse-contracts/commit/f405d0b0e38fabd1048a7a32ad53ec966e4957fb#diff-c6595f0d5324e1295270d210b5ea71c8005443851b8bb92f180ad5c4780ddaba

File: subgraphs/synapse/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+## Total Value Locked USD
+
+Synapse operates by utilizing multiple liquidity pools across many different AMMs. Liquidity providers of these AMMs can stake their LP position with Synapse, to help facilitate cross chain swaps and bridging in return for SYN emissions.
```
Comment (jaimehgb) [inline subgraphs/synapse/README.md:21]: Hmm, I think they have their own AMM. They don’t use third parties’ as far as I know?

File: subgraphs/cbridge/README.md (hunk 1 above)
Comment (ishraq8) [reply subgraphs/cbridge/README.md:11]: sounds good

File: subgraphs/synapse/README.md (hunk 3 above)
Comment (ishraq8) [reply subgraphs/synapse/README.md]: updated to indicate correction: Synapse operates as an AMM utilizing multiple liquidity pools across many different chains. Liquidity providers of the AMM can stake their LP position with Synapse, to help facilitate cross chain swaps and bridging in return for SYN emissions.

File: subgraphs/synapse/README.md (hunk 2 above)
Comment (ishraq8) [reply subgraphs/synapse/README.md:18]: added

**Review by jaimehgb — APPROVED** [review]

**Pull request merged**